db/migrate/20230221082316_create_likes.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
```

Comment:
- [ ] Please 🙏🏽 add the references to the user and the post tables. It represents the relationship between these tables and the like table.